README.md

```diff
@@ -1,4 +1,4 @@
-# nGraph Library [![Build Status][build-status-badge]][build-status]
+# nGraph Library [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Build Status][build-status-badge]][build-status]
```

Comment:
It might be a bit better if we can add something like ![v1.0](https://img.shields.io/badge/1.0-RC1-green.svg) as version.
`![v1.0](https://img.shields.io/badge/1.0-RC1-green.svg)`